drivers/blazer_usb.c

```diff
@@ -360,9 +433,19 @@ static void *phoenix_subdriver(USBDevice_t *device)
 }
 
 
+static void *snr_subdriver(USBDevice_t *device)
+{
+	NUT_UNUSED_VARIABLE(device);
+
+	subdriver_command = &snr_command;
+	return NULL;
+}
+
+
 static usb_device_id_t blazer_usb_id[] = {
 	{ USB_DEVICE(0x05b8, 0x0000), &cypress_subdriver },	/* Agiler UPS */
 	{ USB_DEVICE(0x0001, 0x0000), &krauler_subdriver },	/* Krauler UP-M500VA */
+	{ USB_DEVICE(0x0001, 0x0000), &snr_subdriver },		/* SNR UPS LID600 */
```

Comment:
I am concerned about use of same values as in the previous line to identify the USB device. Alas, some vendors do take shortcuts and/or use same controllers with different firmware, or as seems to be case here - hijack(?)/use a sort of unregistered value.

This may need an end-user to reference explicitly the `snr` subdriver in their `ups.conf` settings.

Which reminds me, docs were not updated by this PR e.g. https://github.com/networkupstools/nut/blob/master/docs/man/blazer-common.txt#L154 and probably nut.dict will sant to accept the new valid word.

Also, if this work is done for a device vendor, you're encouraged to update https://github.com/networkupstools/nut/blob/master/docs/acknowledgements.txt

Reply:
Also, looking at the closed #967 - do not worry about posting the PR'ed change all as one commit, there may be many small meaningful changes to make a feature and PR (and there's rebasing and squashing for small commits like typo fixes) ;)